docker/boot/Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN apk add nginx && \
     mkdir -p /var/cache/nginx && \
     chown -R www:www /var/cache/nginx && \
     chown -R www:www /var/tmp/nginx && \
-    mkdir -p/usr/share/nginx/html/
+    mkdir -p /usr/share/nginx/html/
 
 
```

Comment:
COPY supervisord.conf /etc

This step also doesn't work
It should be COPY files/supervisord.conf /etc